tests/installation/addon_products_sle.pm

```diff
@@ -33,8 +33,9 @@ sub handle_all_packages_medium {
     # According to installation guide, select a sle product is mandatory
     # when install with the all-packages media, so add the base product
     # (sles/sled/etc) as a fake addon
-    push @addons, $sle_prod if !grep(/^$sle_prod$/, @addons);
-
+    unless (is_sle('12-SP2')) {
```

Comment:
What is this condition? =) It's `15-SP2` and we can assume that this behavior will stay in SP3, so let's put
```perl
unless (is_sle('15-SP2+')( {
...
}
```

Also, please add a comment explaining the change.

Reply:
b6b3e0280

Reply:
Looks like it's wrong commit id. Have you forgotten to push to the branch?